services/media/workflows/src/Stations/Movies/MovieGenres/MovieGenres.tsx

```diff
@@ -162,13 +162,14 @@ const Form: React.FC = () => {
 
 const Panel: React.FC<{ data?: MovieGenresQuery }> = ({ data }) => {
   return useMemo(() => {
+    const lastModifiedNode = data?.movieGenres?.nodes?.slice(-1)[0];
```

Comment:
Items in `data?.movieGenres?.nodes` are not ordered according to modified date so you would need to iterate the array to find the last edited item.

Reply:
@axisuru adjusted the pull request thanks for mentioning